pom.xml

```diff
@@ -88,43 +88,66 @@
           </execution>
         </executions>
       </plugin>
-
-      <plugin>
-        <groupId>org.sonatype.plugins</groupId>
-        <artifactId>nexus-staging-maven-plugin</artifactId>
-        <extensions>true</extensions>
-        <configuration>
-          <serverId>rso</serverId>
-          <nexusUrl>https://repository.sonatype.org/</nexusUrl>
-          <autoReleaseAfterClose>true</autoReleaseAfterClose>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <artifactId>maven-release-plugin</artifactId>
-        <configuration>
-          <!-- Something in the pom hierarchy overrides the default 'pushChanges == true', so we reset that here. -->
-          <pushChanges>true</pushChanges>
-          <releaseProfiles>gpg-sign</releaseProfiles>
-          <!--
-          CI build runs out of memory during release, with error like this:
-          Failed to execute goal org.apache.maven.plugins:maven-release-plugin:2.5.2:prepare (default-cli) on project nexus-repository-base: Maven execution failed, exit code: '137' -> [Help 1]
-          TODO: Investigate increasing available memory for the CI build
-          -->
-          <arguments>-DskipTests</arguments>
-        </configuration>
-      </plugin>
     </plugins>
   </build>
 
   <profiles>
     <profile>
-      <id>gpg-sign</id>
+      <id>jenkins-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.sonatype.plugins</groupId>
+            <artifactId>nexus-staging-maven-plugin</artifactId>
+            <version>1.6.8</version>
+            <extensions>true</extensions>
+            <configuration>
+              <autoReleaseAfterClose>true</autoReleaseAfterClose>
+            </configuration>
+          </plugin>
+          <plugin>
+            <artifactId>maven-release-plugin</artifactId>
```

Comment:
It appears the 1maven-release-plugin` is configured the same, in both profiles. Therefore, I think it is simpler to keep it defined in the "main" pom (outside the profiles), but maybe I'm just not seeing the difference.

Also, it appears the builds are still attempting to run the tests during the `release` phase - (causing OOM errors during the release). Maybe this is related to the plugin/profile changes here? Make sure the `labrat2` CircleCI build succeeds in releasing before committing these changes to the community formats builds.